### PR TITLE
add `machine424` and `rexagod`  to OWNERS

### PR DESCRIPTION
**Pull request by rexagod**
Also move Joao to emeritus.

cc @jan--f @machine424

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -8,8 +8,9 @@ reviewers:
   - sthaha
```
Comment (machine424) [inline OWNERS:8]: I think we can excuse Sunil as well.

File: OWNERS [hunk 2]
```diff
@@ -8,8 +8,9 @@ reviewers:
...
   - raptorsun
   - jan--f
-  - JoaoBraveCoding
   - juanrh
```
Comment (simonpasquier) [inline OWNERS:11]: juanrh can be removed too

**Comment by simonpasquier** [comment]
And @marioferh too

**Pull request merged**